netconf_client/connect.py

```diff
@@ -123,14 +145,18 @@ class CallhomeManager:
 
     """
 
-    def __init__(self, bind_to="", port=4334, backlog=1):
+    def __init__(self, bind_to="", port=4334, ipv6=None, backlog=1):
```

Comment:
```suggestion
    def __init__(self, bind_to="", port=4334, ipv6=False, backlog=1):
```

This seems like it wants to be a bool, since that's how it is used later.

Reply:
It is not necessary to be bool type. I'm not check what is exactly the value in further, also it is not required option that has not default value as True.

Reply:
or it should be like `ipv6: bool = False`

Reply:
We can't use that syntax for type annotations because this package still supports Python 2.7 (although we should probably drop that soon).

In general though this should be `False` instead of `None` because it's being used like a bool. It would be very strange for someone to pass any other object type to it.

Reply:
Sorry for my late response.
I suggest appending new parameters to the end of the parameter list for backward compatibility for users who do not use named arguments but rely on the known parameter sequence.

Reply:
Fixed: `(self, bind_to="", port=4334, backlog=1, ipv6=False)`